abuild.in

```diff
@@ -1728,6 +1743,28 @@ dev() {
 	default_dev
 }
 
+# predefined splitfunc static
+default_static() {
+	local i=
+	depends=""
+	pkgdesc="$pkgdesc (static library)"
+
+	cd "$pkgdir" || return 0
+	local libdirs=usr/
```

Comment:
Should this perhaps be `usr/lib/`, or are we expecting static libraries outside of `/usr/lib` but still within `/usr`?

Reply:
yes it should be /usr/lib, error on my part.